src/containers/profile/Profile.js

```diff
@@ -1,27 +1,55 @@
 import React, { useState, useEffect } from "react";
 import GithubProfileCard from "../../components/githubProfileCard/GithubProfileCard";
-import axios from "axios";
+import ApolloClient from "apollo-boost";
+import { gql } from "apollo-boost";
 import { openSource } from "../../portfolio";
 import Contact from "../contact/Contact";
 
-
 export default function Profile() {
   const [prof, setrepo] = useState([]);
   function setProfileFunction(array) {
     setrepo(array);
   }
   function getProfileData() {
-    const client=axios.get("https://api.github.com/users/"+openSource.githubUserName)
-      .then(function(response){
-            setProfileFunction(response.data);
+    const client = new ApolloClient({
+      uri: "https://api.github.com/graphql",
+      request: (operation) => {
+        operation.setContext({
+          headers: {
+            authorization: `Bearer ${openSource.githubConvertedToken}`,
+          },
+        });
+      },
     });
+
+    client
+      .query({
+        query: gql`
+      {
+        user(login:"${openSource.githubUserName}") { 
+          name
+          bio
+          isHireable
+          avatarUrl
+          location
+        }
+    }
+      `,
+      })
+      .then((result) => {
+        setProfileFunction(result.data.user);
+      });
```

Comment:
Add an error handler here.

Reply:
Basically add these lines here
```suggestion
      }
      .catch(function (error) {
        console.log(error);
      };
```